Source/Core/DolphinWX/MemcardManager.cpp

```diff
@@ -430,7 +430,7 @@ bool CMemcardManager::CopyDeleteSwitch(u32 error, int slot)
 		if (slot != -1)
 		{
 			memoryCard[slot]->FixChecksums();
-			if (!memoryCard[slot]->Save()) PanicAlertT(E_SAVEFAILED);
+			if (!memoryCard[slot]->Save()) PanicAlert(E_SAVEFAILED);
```

Comment:
Oh, okay. I thought translation wouldn't work because it wasn't a regular string literal. I'll change it back.